docs/sphinx/dev/dev_style_guide.rst

```diff
@@ -26,9 +31,12 @@ Detailed Rules
 File Organization
 ------------------
 
-* ``kotekanProcess`` files should be placed in the **lib/processes/** folder, and named in *camelCase* to match the process class they contain.
+* ``kotekanProcess`` files should be placed in the **lib/processes/** folder,
+and named in *CamelCase* to match the process class they contain.
 
-* Accelerator interfaces (such as e.g. OpenCL or CUDA) should create their own folder within **lib/**, for storage of kernel handling objects and the kernels themselves.
+* Accelerator interfaces (such as e.g. OpenCL or CUDA) should create their own
+folder within **lib/**, for storage of kernel handling objects and the kernels
+themselves.
 
 * Support scripts and components can be placed in the **script/** folder.
```

Comment:
I'd add that Python components/scripts should be going to the `python/` dir if reasonable.